rules/scripts/bitstreams_workspace.py

```diff
@@ -279,30 +351,34 @@ def _ConstructBazelString(self, build_file: Path, key: str) -> str:
         if key == 'latest':
             key = self.available['latest']
 
-        files_by_extension = self.GetFromCache(key)
-
-        if len(files_by_extension.get('orig', set())) != 1 or \
-           len(files_by_extension.get('splice', set())) != 1:
-            error_msg_lines = [
-                "Could not find the bitstreams to generate a BUILD file:" +
-                repr(build_file),
-                "in files_by_extension:" + repr(files_by_extension),
-                "using key:" + repr(key),
-            ]
-            logging.error('\n'.join(error_msg_lines))
-            sys.exit(1)
-
-        (test_rom_file, ) = files_by_extension['orig']
-        (rom_file, ) = files_by_extension['splice']
-
-        def filegroup_lines(name, src):
-            return [
-                'filegroup(',
-                '    name = "{}",'.format(name),
-                '    srcs = ["{}"],'.format(src),
-                ')',
-                '',
-            ]
+        manifest = self.GetFromCache(key)
+        designs = dict()
+        if manifest["schemaVersion"] > 1:
+            logging.warning("Warning: Manifest is newer than available schemas")
+            logging.warning("Will try parsing an available schema with highest version")
+            manifest["schemaVersion"] = 1
```

Comment:
Nit: should the use of the version `1` here be a constant defined at the top of this file?
```
MANIFEST_SCHEMA_VERSION = 1
```

I think the if block below can continue using literals, as that is expected to grow into something like:
```python
if manifest["schemaVersion"] == 1:
   # version 1 stuff
elif manifest["schemaVersion"] == 2:
   # version 2 stuff
elif manifest["schemaVersion"] == 3:
   # version 3 stuff
else:
    logging.error("Bad manifest version {}", manifest["schemaVersion"])

Reply:
Done.